test/allocators/AllocatorWithManager.sol.js

```diff
@@ -363,54 +456,122 @@ describe('AllocatorWithManager', function () {
     });
     context('Slot has not been set before', function () {
       it('does nothing', async function () {
+        const { roles, allocatorWithManager, slotIndex } = await helpers.loadFixture(deploy);
         await expect(
           allocatorWithManager.connect(roles.randomPerson).resetSlot(roles.airnode.address, slotIndex)
         ).to.not.emit(allocatorWithManager, 'ResetSlot');
       });
     });
   });
 
-  describe('setterOfSlotIsCanStillSet', function () {
-    context('Setter of slot is still a slot setter', function () {
+  describe('slotCanBeResetByAccount', function () {
+    context('Slot is set by the account', function () {
       it('returns true', async function () {
+        const { roles, allocatorWithManager, slotIndex, expirationTimestamp, subscriptionId } =
+          await helpers.loadFixture(deploy);
         await allocatorWithManager
           .connect(roles.slotSetter)
           .setSlot(roles.airnode.address, slotIndex, subscriptionId, expirationTimestamp);
-        expect(await allocatorWithManager.setterOfSlotIsCanStillSet(roles.airnode.address, slotIndex)).to.equal(true);
+        expect(
+          await allocatorWithManager.slotCanBeResetByAccount(roles.airnode.address, slotIndex, roles.slotSetter.address)
+        ).to.equal(true);
       });
     });
-    context('Setter of slot is the manager address', function () {
-      it('returns true', async function () {
-        await allocatorWithManager
-          .connect(roles.manager)
-          .setSlot(roles.airnode.address, slotIndex, subscriptionId, expirationTimestamp);
-        expect(await allocatorWithManager.setterOfSlotIsCanStillSet(roles.airnode.address, slotIndex)).to.equal(true);
+    context('Slot is not set by the account', function () {
+      context('Slot has not expired', function () {
+        context('Setter of the slot is the manager', function () {
+          it('returns false', async function () {
+            const { roles, allocatorWithManager, slotIndex, expirationTimestamp, subscriptionId } =
+              await helpers.loadFixture(deploy);
+            await allocatorWithManager
+              .connect(roles.manager)
+              .setSlot(roles.airnode.address, slotIndex, subscriptionId, expirationTimestamp);
+            expect(
+              await allocatorWithManager.slotCanBeResetByAccount(
+                roles.airnode.address,
+                slotIndex,
+                roles.randomPerson.address
+              )
+            ).to.equal(false);
+          });
+        });
+        context('Setter of the slot is still a slot setter', function () {
```

Comment:
```suggestion
        context('Setter of the slot is a slot setter', function () {
```
Kind of confused by "still" 🤔

Reply:
For them to have set a slot, they must have been a slot setter at one point. So I wanted to emphasize the distinction between someone who's not a slot setter and someone who was a slot setter at some point that set a slot and is no longer a slot setter.